medical_procedure_external/views/medical_encounter.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2022 CreuBlanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="medical_encounter_form_view">
+        <field
+            name="name"
+        >medical.encounter.form (in medical_procedure_external)</field>
+        <field name="model">medical.encounter</field>
+        <field
+            name="inherit_id"
+            ref="medical_administration_encounter.medical_encounter_view_form"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//header/button[last()]" position="after">
+                <button
+                    type="action"
+                    name="%(medical_procedure_external.medical_encounter_create_procedure_external_act_window)s"
+                    class="oe_stat_button btn-warning"
+                    icon="fa-file-prescription"
+                    string="Generate Request"
```

Comment:
I think it is best to call it "Generate External Request", in order to not confuse it with internal requests

Reply:
Donee!!